bug(import): If the data source has a string type and contains the character ',' then import will panic

recurrent:

```shell
$ pwd
/home/lianxm/github.com/oomstore/oomcli/test
$ cat user_click.csv
user,unix_milli,last_5_click_posts,number_of_user_starred_posts
1,1644285708319,"1,2",10
$ oomcli import \
    --group user-click \
    --input-file "$(pwd)/user_click.csv" \
    --description 'test data' > /dev/null

panic: runtime error: index out of range [4] with length 4

goroutine 1 [running]:
github.com/oom-ai/oomstore/internal/database/dbutil.ReadLine({0xc00081e450, {0xc0001222e8, 0x4}, {0xc000348180, 0x4, 0x4}, {0xc000119610, 0x2, 0x2}})
        /home/lianxm/github.com/oomstore/internal/database/dbutil/data_loader.go:78 +0x611
github.com/oom-ai/oomstore/pkg/oomstore.(*OomStore).csvReaderImportStream(0x7ffcef2e91de, {0x1da24a8, 0xc000122000}, 0xc000348000, 0xc00081fc20)
        /home/lianxm/github.com/oomstore/pkg/oomstore/import_stream.go:40 +0x4d0
github.com/oom-ai/oomstore/pkg/oomstore.(*OomStore).Import(0x0, {0x1da24a8, 0xc000122000}, {{0x7ffcef2e91c6, 0xa}, {0x7ffcef2e9228, 0x9}, 0x0, 0x0, 0x2b2ee60, ...})
        /home/lianxm/github.com/oomstore/pkg/oomstore/import.go:35 +0x433
github.com/oom-ai/oomstore/oomcli/cmd.glob..func21(0x2b1c6c0, {0x1aa37a5, 0x6, 0x6})
        /home/lianxm/github.com/oomstore/oomcli/cmd/import.go:42 +0x165
github.com/spf13/cobra.(*Command).execute(0x2b1c6c0, {0xc000455980, 0x6, 0x6})
        /home/lianxm/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x2b1dac0)
        /home/lianxm/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/lianxm/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:902
github.com/oom-ai/oomstore/oomcli/cmd.Execute()
        /home/lianxm/github.com/oomstore/oomcli/cmd/root.go:32 +0x25
main.main()
        /home/lianxm/github.com/oomstore/oomcli/main.go:6 +0x17
```

--- a/internal/database/dbutil/data_loader.go
+++ b/internal/database/dbutil/data_loader.go
@@ -2,8 +2,8 @@ package dbutil
 
 import (
 	"context"
+	"encoding/csv"
 	"io"
-	"strings"
 
 	"github.com/jmoiron/sqlx"
 	"github.com/oom-ai/oomstore/pkg/errdefs"
@@ -67,14 +67,19 @@ type ReadLineOpt struct {
 	Features   types.FeatureList
 }
 
+// ReadLine read a line from data source
 func ReadLine(opt ReadLineOpt) ([]interface{}, error) {
-	row, err := opt.Source.Reader.ReadString('\n')
+	// Currently, oomstore only supports csv format data source,
+	// so here we use csv.Reader to read data directly.
+	// If there are other data formats in the future, we need to choose different decoders according to the different formats.
+	reader := csv.NewReader(opt.Source.Reader)
+	row, err := reader.Read()
 	if err != nil {
-		return nil, errdefs.WithStack(err)
+		return nil, err
 	}
-	rowSlice := strings.Split(strings.Trim(row, "\n"), opt.Source.Delimiter)
-	line := make([]interface{}, 0, len(rowSlice))
-	for i, ele := range rowSlice {
+
+	line := make([]interface{}, 0, len(row))
+	for i, ele := range row {
 		if len(opt.Header) == 0 || len(opt.Features) == 0 || opt.Header[i] == opt.EntityName {
 			// entity_key doesn't need to change type
 			line = append(line, ele)

--- a/oomcli/test/test_import_csv_file.sh
+++ b/oomcli/test/test_import_csv_file.sh
@@ -5,10 +5,19 @@ source ./util.sh
 init_store
 register_features
 
-case='oomcli import using absolute path'
+case='oomcli import batch feature using absolute path'
 oomcli import \
     --group phone \
     --input-file "$(pwd)/device.csv" \
+    --description 'test data' > /dev/null
+actual=$?
+expected=0
+assert_eq "$case" "$expected" "$actual"
+
+case='oomcli import stream feature using absolute path'
+oomcli import \
+    --group user-click \
+    --input-file "$(pwd)/user_click.csv" \
     --description 'test data' > /dev/null
 actual=$?
 expected=0

--- a/oomcli/test/user_click.csv
+++ b/oomcli/test/user_click.csv
@@ -1,0 +1,3 @@
+user,unix_milli,last_5_click_posts,number_of_user_starred_posts
+1,1644285708319,"1,2",10
+2,1644285708319,"2,3",10

